remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerImpl.java

```diff
@@ -30,56 +40,114 @@
 import org.infinispan.query.remote.client.ProtobufMetadataManagerConstants;
 import org.infinispan.query.remote.client.impl.MarshallerRegistration;
 import org.infinispan.query.remote.impl.indexing.IndexingMetadata;
+import org.infinispan.registry.InternalCacheRegistry;
+import org.infinispan.security.AuthorizationPermission;
+import org.infinispan.security.impl.CacheRoleImpl;
+import org.infinispan.transaction.LockingMode;
+import org.infinispan.transaction.TransactionMode;
+import org.infinispan.util.concurrent.IsolationLevel;
 
 /**
  * @author anistor@redhat.com
  * @since 7.0
  */
 @MBean(objectName = ProtobufMetadataManagerConstants.OBJECT_NAME,
-      description = "Component that acts as a manager and container for Protocol Buffers message type definitions in the scope of a CacheManger.")
+      description = "Component that acts as a manager and persistent container for Protocol Buffers message type definitions in the scope of a CacheManger.")
 @Scope(Scopes.GLOBAL)
 public final class ProtobufMetadataManagerImpl implements ProtobufMetadataManager {
 
-   private volatile Cache<String, String> protobufSchemaCache;
+   private Cache<String, String> protobufSchemaCache;
 
    private ObjectName objectName;
 
    private final SerializationContext serCtx;
 
-   @Inject EmbeddedCacheManager cacheManager;
+   @Inject
+   EmbeddedCacheManager cacheManager;
+
+   @Inject
+   InternalCacheRegistry internalCacheRegistry;
```

Comment:
It's a lot more compact if you put the `@Inject` on the same line and don't leave an empty line between fields.

Reply:
-1